doc/Caliptra.md

```diff
@@ -1255,47 +1263,45 @@ Caliptra provides a HW API to do a SHA384 hash calculation. The SoC can access t
 
 ![](./images/Caliptra_boot_flow6.png)
 
-*Figure 4: Debug Flow*
-
 1. SOC will drive the security state indicating that its a debug flow. Refer to “Caliptra Security States” for encodings.
 2. SOC (using a GPIO pin or SOC ROM) will drive BootFSMBrk (this is also used for debug cases). This can be driven at any time before cptra\_rst\_b is deasserted.
 3. SOC will follow the boot flow as defined in Caliptra IP HW boot flow to assert cptra\_pwrgood and deassert cptra\_rst\_b, followed by writing to the fuse registers.
-4. HVM through JTAG or using Caliptra SOC interface writes to “CPTRA\_DBG\_MANUF\_SERVICE\_REG” requesting for appropriate debug service (refer to the ROM/FW specification for bit definitions)
-5. HVM through JTAG can also inject uController TAP commands at this point following the [Veer specification](https://github.com/chipsalliance/Cores-VeeR-EL2)
+4. HVM through JTAG or using Caliptra SOC interface writes to “CPTRA\_DBG\_MANUF\_SERVICE\_REG” requesting for appropriate debug service (refer to the ROM/FW spec for bit definitions)
```

Comment:
a previous commit changed all usage of "spec" to "specification"

Reply:
fixed